### DISCUSSION TITLE
Home Command Permission with a limit for N homes

### DISCUSSION THREAD

**Discussion by NovaPixell** [EssentialsX/Essentials#4092] [Ideas and Feature Suggestions] [answered]
For example:
essentials.home.use.2 = The player can use 2 home (ex. bed and home1)

For example:
essentials.home.access.4 =  The player can use 4 home

I really need this on my server :D

**Comment by triagonal** [comment, marked as answer]
There probably isn't ever going to be a limit put on how many homes a player can access, but there's already a system for allowing multiple homes to be set, if that works? Look at the `sethome-multiple` section in your config. You can read more about how to set it up [here](https://wiki.mc-ess.net/wiki/Multihome).

**Reply by NovaPixell** [reply]
> There probably isn't ever going to be a limit put on how many homes a player can access, but there's already a system for allowing multiple homes to be set, if that works? Look at the `sethome-multiple` section in your config. You can read more about how to set it up [here](https://wiki.mc-ess.net/wiki/Multihome).

Exactly. but now I explain. 
On my server we assign temporary ranks.

for example rank 1 gives access to 2 homes for 30 days.
rank 2 gives access to 3 homes for 30 days.

Essentially, if a user decides to be Rank 2 for one month and then the next Rank 1, he will always have access to 3 home. But it really should have 2. I hope I have explained

**Reply by triagonal** [reply]
As explained by @mdcfe in #3977, this isn't something that's likely to be added to EssentialsX.
